Source/multi.cpp

```diff
@@ -662,8 +662,10 @@ void multi_process_network_packets()
 		}
 		HandleAllPackets(playerId, (const byte *)(pkt + 1), dwMsgSize - sizeof(TPktHdr));
 	}
+#ifndef PSP
 	if (SErrGetLastError() != STORM_ERROR_NO_MESSAGES_WAITING)

```

Comment:
Not exactly sure why the port is choking on this one, but we should probably investigate it a bit before just ignoring it like it did here.

Reply:
This should be fixed now and the workaround can be removed.